src/test/java/com/twilio/jwt/validation/ValidationTokenTest.java

```diff
@@ -104,29 +107,29 @@ public void testTokenFromHttpRequest() throws IOException {
 
         this.validateToken(claims);
         Assert.assertEquals("authorization;host", claims.get("hrh"));
-        Assert.assertEquals("4b3d2666845a38f00259a5231a08765bb2d12564bc4469fd5b2816204c588967", claims.get("rqh"));
+        Assert.assertEquals("712fbbec9dcb4fe58ed8caecf925d2fe10889f5d3f4b48e748157a4a1113697d", claims.get("rqh"));
     }
 
     @Test
-    public void testTokenFromPostRequest() throws IOException {
+    public void testTokenFromEntityRequest() throws IOException {
```

Comment:
it seems like the use case we are targeting in this pr, `POST requests that are void of a body or any params` is missing testing conditions here

Reply:
It's captured in the previous test case (L88) which I modified so that it would fail at first, rather than creating an additional test case for similar scenarios.